src/content/app-platform/overview/index.md

```diff
@@ -30,6 +31,9 @@ The _Giant Swarm App Platform_ refers to a set of features and concepts that all
 you to browse, install and manage the configurations of apps (such as Prometheus)
 from a single place; the management cluster.
 
+We fully support [Helm](https://helm.sh/) as a general tool to deploy your applications as well as for our general App Catalog. Apps are packaged as Helm charts and can be configured with _values_. We provide a recommended
+[app configuration]({{< relref "/app-platform/app-configuration" >}}) which you can override to meet your needs.
```

Comment:
This is based on https://docs.giantswarm.io/app-platform/app-configuration/ which I think maybe @pipo02mix wrote?

Currently, the overview doesn't mention helm at all which I think is confusing. I hope this better explains how we build on top of helm. WDYT?

Reply:
I do think was @oponder but I like it anyway :)